case_studies/lowell_high_school/03_second_meeting/recap.md

```diff
@@ -0,0 +1,15 @@
+Overall the meeting went well! We did not do the ice breaker game, because so
+many people were coming in late or straggling, but this time we had more 
+attendees, around 12 I believe. The majority of them seemed to like the 
+workshop, which was awesome. Many finished building their portfolio webpage in
+about 30-45 minutes.
+
+Getting the attendees to talk was one of my primary concerns, but once I 
+talked to them about how important asking questions is, and how you should 
+rely on the people sitting next to you and the internet for help, many seemed
+to want to help each other when they finished, or they worked together to 
+figure out a problem. 
+
+Going forward, we are still trying to recruit more people. 12 is a solid
+amount, but I doubt all 12 will show up each week. We are trying to put up more
+posters to get our name out there. 
```

Comment:
@temucher If I understand correctly, you need to apply to be a club at lowell (instead of a group) to put  up signs. What do you have to do to get access to advertising?